### PR TITLE
Add a package.json to enable people to include it as an npm package

### PR DESCRIPTION
**Pull request by jjoos**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "jquery.textarea-helper",
```
Comment (amasad) [inline package.json]: can we just call it `textarea-helper`? seems to be available: https://www.npmjs.com/package/textarea-helper

Comment (jjoos) [inline package.json]: done!

**Comment by jjoos** [comment]
@amasad ping

**Comment by amasad** [comment]
Thanks published textarea-helper@0.3.1

**Pull request merged**